app/gzac/src/main/resources/config/zgw/bezwaar.zgw-document-upload-fields.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Is this the format of the field you envision? i would think that the fields should be in the same array since they are for the same case definition.

Reply:
This is the format described in the solution design